### ISSUE TITLE
[BUG] Function plot_series raising Exception when having pred_interval for a time serie of length 3

### ISSUE THREAD

**Issue by admo1** [sktime/sktime#5493]
**Describe the bug**
Function plot_series is raising an Exception when having pred_interval for a time serie of length 3.
The Exception is the following :
- ValueError: operands could not be broadcast together with shapes (x,x)

**To Reproduce**

```python
from sktime.utils.plotting import plot_series
from sktime.datasets import load_airline
from sktime.forecasting.naive import NaiveForecaster
from sktime.forecasting.base import ForecastingHorizon
model = NaiveForecaster()
serie = load_airline()
model.fit(serie[:-3])
fhorizon = ForecastingHorizon(serie[-3:].index, is_relative=False)
pred = model.predict(fhorizon)
interval = model.predict_interval(fhorizon)
plot_series(serie[:-3], serie[-3:], pred, pred_interval=interval)
```

**Expected behavior**
The function should not raise any exception and show the prediction interval as expected.

**Additional context**
I also don't understand why the time serie is not showing a lineplot when we have 3 or less points.
I would like to be able to disable this behavior.

**Versions**
<details>

System:
    python: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
executable: /local_disk0/.ephemeral_nfs/envs/pythonEnv-99676664-f42e-4b1d-b8b4-cb3cdc5cf19c/bin/python
   machine: Linux-5.15.0-1049-azure-x86_64-with-glibc2.35

Python dependencies:
          pip: 22.2.2
       sktime: 0.24.0
      sklearn: 1.1.1
       skbase: 0.6.1
        numpy: 1.22.4
        scipy: 1.9.1
       pandas: 1.4.4
   matplotlib: 3.5.2
       joblib: 1.2.0
        numba: 0.58.1
  statsmodels: 0.13.2
     pmdarima: 2.0.4
statsforecast: 1.6.0
      tsfresh: 0.20.1
      tslearn: 0.5.3.2
        torch: None
   tensorflow: 2.13.1
tensorflow_probability: None

</details>

**Comment by fkiraly** [comment]
Well, that is certainly odd.

Bug is confirmed for 3 or less points, and is not present for 4 or more.

**Comment by fkiraly** [comment]
fix is in #5494, for review
